packages/create-cloudflare/src/helpers/latestPackageVersion.ts

```diff
@@ -0,0 +1,14 @@
+import { fetch } from "undici";
+
+interface NpmInfoResponse {
+	"dist-tags": { latest: string };
+}
```

Comment:
just out of curiosity, any reason in particular to use an `interface` here instead of a `type`? 🙂

Reply:
Just did that without thinking. There is basically no difference in this scenario, right?

Reply:
Yes I don't think there's really a difference here, I was just thinking of consistency thought the codebase (I think that in C3 `type` is generally used)

![Screenshot 2024-01-18 at 16 00 31](https://github.com/cloudflare/workers-sdk/assets/61631103/db618955-6a81-4667-b58e-753bc33d0ef9)
![Screenshot 2024-01-18 at 16 00 43](https://github.com/cloudflare/workers-sdk/assets/61631103/107f33c8-2c64-47a1-b1eb-98a21b4e4a64)

But I am fine with either 🙂👍

Reply:
as I said I was just curious, I leave it up to you 😄

Reply:
Done